eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -3,6 +3,9 @@
 
 <UsageData>
   <IgnorePatterns>
+    <Usage Id="Microsoft.DotNet.XliffTasks" />
```

Comment:
This and the sourcelink baseline additions should not be necessary. Add the following dependencies to the eng/Version.Details.xml file under the ToolsetDependencies section: 

https://github.com/dotnet/sdk/blob/main/eng/Version.Details.xml#L422-L426
https://github.com/dotnet/aspnetcore/blob/main/eng/Version.Details.xml#L376-L380

Before sure to run `darc update-dependencies --coherency-only` after doing so.